extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -941,17 +946,18 @@ export class SchemaCompareMainWindow {
 				return;
 			}
 
+			let ownerUri;
```

Comment:
the logic for Source and Target are highly similar; can we combine them into two calls to a helper function?

```
this.sourceEndpointInfo = constructEndpointInfo(result.sourceEndpointInfo, SourceTitle);
this.targetEndpointinfo = constructEndpointInfo(result.targetEndpointInfo, TargetTitle);
```

Reply:
That makes sense. Created a helper function.